cmake/CheckDGtalDependencies.cmake

```diff
@@ -14,9 +14,17 @@ set(Boost_USE_STATIC_RUNTIME OFF)
 set(Boost_FOUND FALSE)
 FIND_PACKAGE(Boost 1.46.0 REQUIRED)
 if ( Boost_FOUND )
-  message(STATUS "Boost found.")
   include_directories( ${Boost_INCLUDE_DIRS} )
   SET(DGtalLibInc ${DGtalLibInc} ${Boost_INCLUDE_DIRS})
+
+  ## Checking boost/random ( <1.47)
+  STRING (COMPARE LESS "${Boost_MINOR_VERSION}" 47 BOOST_RANDOM_OLD)
```

Comment:
What will happen when Boost reaches version 2 (one day) ? Isn't it safer to use `BOOST_VERSION` available in `FindBoost`:
```STRING (COMPARE LESS "${Boost_VERSION}" 104700 BOOST_RANDOM_OLD)```